### DISCUSSION TITLE
allure-report folder is not found in current working directory on Azure VM

### DISCUSSION THREAD

**Discussion by abhikhanzode** [allure-framework/allure2#1630] [Bug Reports] [answered]
Hello Allure Team, 

We are trying to publish an allure report on WebServer via the Azure DevOps pipeline. and to host the allure report on the webserver we need an HTML file that can generate by using the _"allure generate allure-results"_ command. 
But when we are trying to generate an allure report from Azure DevOps Microsoft hosted pipeline. The allure-report folder is not found in the current working directory. 

Can you provide any solution to it? 

Thanks 
Abhijeet

**Comment by epszaw** [comment, marked as answer]
Try to use our [official extension](https://marketplace.visualstudio.com/items?itemName=qameta.allure-azure-pipelines) for Azure DevOps. It does all these things out of the box.
